.github/workflows/build.yaml

```
@@ -87,7 +107,7 @@ jobs:
       date: ${{ inputs.date }}
       package-name: pylibcudf
   wheel-build-cudf:
-    needs: wheel-publish-pylibcudf
+    needs: [wheel-publish-libcudf, wheel-publish-pylibcudf]
```

We can probably remove this, as it's a transitive requirement (libcudf -> pylibcudf -> cudf).
```suggestion
    needs: wheel-publish-pylibcudf
```

---

sure, fine with me